source/tests/pt/test_hybrid_muon.py

```
@@ -0,0 +1,231 @@
+            torch.nn.ReLU(),
+            torch.nn.Linear(20, 5, device=self.device),
+        )
+        optimizer = HybridMuonOptimizer(model.parameters(), lr=0.02)
+
+        x = torch.randn(4, 10, device=self.device)
+        model(x).sum().backward()
+
+        initial_params = [p.clone() for p in model.parameters()]
+        optimizer.step()
+
+        for i, (p, init_p) in enumerate(zip(model.parameters(), initial_params)):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)